features/step_definitions/artefact_steps.rb

```diff
@@ -161,3 +172,26 @@
 When /^I try to create a new artefact with the same need$/ do
   visit new_artefact_path(:artefact => {:need_id => @artefact.need_ids.first})
 end
+
+When /^I browse to the "(.*)" URL for the artefact$/ do |path|
+  visit archive_artefact_path(@artefact)
+end
+
+And /^I click the archive tab on the artefact page$/ do
+  visit edit_artefact_path(@artefact)
+  click_link "Archive"
+end
+
+And /^I archive the existing artefact$/ do
+  Artefact.observers.disable :update_search_observer do
+    @artefact.update_attributes!('state' => 'archived')
```

Comment:
This step should use the interface to archive the object by clicking an appropriate link or button